core/src/main/java/com/amplifyframework/datastore/DataStoreCategoryBehavior.java

```diff
@@ -227,7 +227,29 @@ void observe(
     );
 
     /**
-     * Resets the underlying DataStore to its pre-initialized state such that no data remains on the local
+     * Starts the DataStore.  This only needs to be called if you wish to start eagerly.  If you don't call it,
+     * it will be called automatically prior to executing any other operations (#query, #save, #delete, #observe).
```

Comment:
Suggest a few tweaks to language to:
1. Remove use of "it", which increases cognitive buffering in dense technical writing, and
2. Talk explicitly about synchronization with a remote system. Currently, there is not enough differentiation from `configure()` or `initialize()`. 
```suggestion
     * Starts the DataStore's synchronization with a remote system, if DataStore is configured to support
     * remote synchronization. This only needs to be called if you wish to start the synchronization eagerly.
     * If you don't call start(), the synchronization will start automatically, prior to executing any other
     * operations (#query, #save, #delete, #observe).
```

I might suggest going back and updating `configure()`, `initialize()` (well, it's on the category), and `clear()`, to more explicitly highlight their relationship to the new `start()` and `stop()`.